main.tf

```diff
@@ -1,3 +1,3 @@
-provider "newrelic" {
-  
-}
+module "newrelic" {
+  source = "./newrelic"
```

Comment:
Is moving everything under a module just to make the top-level directory cleaner? If so, I wouldn't bother with the top-level module and just have the subdirectory be the Terraform root.